packages/block-library/src/navigation/index.php

```diff
@@ -334,12 +349,7 @@ function block_core_navigation_get_fallback_blocks() {
 
 	// If `core/page-list` is not registered then return empty blocks.
 	$fallback_blocks = $registry->is_registered( 'core/page-list' ) ? $page_list_fallback : array();
-
-	if ( class_exists( 'WP_Navigation_Fallback' ) ) {
-		$navigation_post = WP_Navigation_Fallback::get_fallback();
-	} else {
-		$navigation_post = Gutenberg_Navigation_Fallback::get_fallback();
-	}
+	$navigation_post = block_core_navigation_get_fallback_post();
```

Comment:
This is the only place this function is now called; let's maybe re-inline the code? 😊

Reply:
Done in https://github.com/WordPress/gutenberg/pull/57754/commits/f8a9a4f97241caa429586c3abd0a0ad454e2f7d8